src/util.c

```diff
@@ -1334,7 +1335,42 @@ static void test_fixedpoint_d2string(void) {
     assert(sz == 0);
 }
 
-#define UNUSED(x) (void)(x)
+#if defined(__linux__) && defined(HAVE_FADVISE)
```

Comment:
maybe we should drop the HAVE_FADVISE condition?
this will server as an assertion that on linux we did detect that FADVISE is supported.

Reply:
Make sense